parcels/interaction/base_hash.py

```diff
@@ -0,0 +1,167 @@
+from abc import ABC, abstractmethod
+
+import numpy as np
+
+
+class BaseHashNeighborSearch(ABC):
```

Comment:
I'm missing here some standard operators, e.g. `__init__` and `__del__` - or does this abstract class define no member variables or allocate resources ? Just curious.

Reply:
The `__init__` of the base class (or derived class are used instead). I haven't defined any `__del__` operator, but I'm also not sure what the purpose would be in this case, since everything should be automatically clean up after the instance goes out of scope. Maybe you can elaborate?